cds/modules/previewer/api.py

```diff
@@ -90,12 +89,8 @@ def record_uri(self):
     @property
     def embed_uri(self):
         """Get the embed uri."""
-        return url_for(
-            'invenio_records_ui.recid_preview',
-            pid_value=self.pid.pid_value,
-            filename=self.file.key,
-            _external=True
-        )
+        rn = self.record.get('report_number', [''])[0]
+        return "{domain}video/{rn}".format(domain=request.url_root, rn=rn)
```

Comment:
can we re-use any other code instead of hardcoding it like this?
any better ideas?

Reply:
`url_for('cds_redirector.video_embed_alias', report_number=rn, _external=True)` maybe?

Reply:
Great, thanks! Fixed!